### PR TITLE
Re-add plugin configuration for dd4hep

### PR DESCRIPTION
**Pull request by vvolkl**

### CODE REVIEW

File: config/packages.yaml [hunk 1]
```diff
@@ -145,6 +145,13 @@ packages:
     target: []
     providers: {}
     compiler: []
+  dd4hep:
+    variants: +lcio+xercesc+hepmc3+geant4
```
Comment (andresailer) [inline config/packages.yaml:149]:
```suggestion
    variants: +lcio+xercesc+hepmc3+geant4+edm4hep
```

Comment (vvolkl) [reply config/packages.yaml:149]: I set it to be enabled  by default, but yes, it's good to add it explicitly

**Comment by vvolkl** [comment]
(There is currently some duplication between the `spack.yaml`, which is specific to the environment and the global `packages.yaml`. I'll switch to using environments completely if there is no issue. )

**Review by andresailer — APPROVED** [review]
Thanks!

**Comment by vvolkl** [comment]
Also adding assimp, since there was some interest in FCC in working with CAD files.

**Review by fdplacido — APPROVED** [review]
Great, thanks!

**Pull request merged**